polkadot/runtime/parachains/src/assigner_coretime/mod.rs

```diff
@@ -322,6 +322,14 @@ impl<T: Config> AssignmentProvider<BlockNumberFor<T>> for Pallet<T> {
 		let config = configuration::ActiveConfig::<T>::get();
 		config.scheduler_params.num_cores
 	}
+
+	fn assignment_duplicated(assignment: &Assignment) {
```

Comment:
Looking forward to when we have this API properly fixed.

Reply:
🙏🏻 tracked by https://github.com/paritytech/polkadot-sdk/issues/5529